snuba/migrations/snuba_migrations/stacktraces/0001_stacktraces.py

```diff
@@ -0,0 +1,92 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import (
+    UUID,
+    Column,
+    DateTime,
+    String,
+    UInt,
+    NamedTuple,
+)
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+columns: List[Column[Modifiers]] = [
+    # primary key
+    Column("organization_id", UInt(64)),
+    Column("project_id", UInt(64)),
+    Column("transaction_id", UUID()),
+    Column("received", DateTime()),
```

Comment:
Yes. For more context, `start_time` indicated when the trace measurement started, `received` is more about when the measurement was done or even when we received this trace.

All that matters is we have a timestamp to let us order by time and aggregate more recent stack traces instead of old ones.